src/__tests__/posthog-core.js

```diff
@@ -677,24 +678,121 @@ describe('_loaded()', () => {
         })
     })
 
-    it('captures pageview', () => {
-        given('config', () => ({
-            capture_pageview: true,
-        }))
+    describe('capturing pageviews', () => {
+        beforeEach(() => {
+            given('config', () => ({
```

Comment:
This does not need to be in a beforeEach. See also https://github.com/tatyshev/given2#more-examples